modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ClusterConfigurationModelIndexer.java

```diff
@@ -53,7 +68,7 @@ public void initialize() throws PortletException {
 			}
 
 			if (_clusterMasterExecutor.isMaster()) {
-				_bundleTracker = _configurationModelIndexer.initialize();
+				_bundleTracker = _initialize();
```

Comment:
Hey @Valmir-unicap, please initialize the `_bundleTracker` into the `_initialize` method and make the method `void`. Thx